src/bindings/python/tests/test_runtime/test_compiled_model.py

```diff
@@ -175,7 +179,8 @@ def test_infer_new_request_tensor_numpy_copy(device):
     res_tensor = compiled_model.infer_new_request({"data": tensor})
     res_img = compiled_model.infer_new_request({"data": img})
     assert np.argmax(res_tensor[list(res_tensor)[0]]) == 531
-    assert np.argmax(res_tensor[list(res_tensor)[0]]) == np.argmax(res_img[list(res_img)[0]])
+    assert np.argmax(res_tensor[list(res_tensor)[0]]) == np.argmax(
```

Comment:
Could you please roll back all changes which don't connected with properties replacement